src/views/ConsentContinuationOut.vue

```diff
@@ -5,14 +5,6 @@
       attach="#consent-continuation-out"
     />
 
-    <GenericErrorDialog
-      attach="#consent-continuation-out"
-      :dialog="courtOrderDialog"
-      :text="courtOrderDialogText"
-      :title="courtOrderDialogTitle"
-      @close="courtOrderDialog=false; goToDashboard()"
-    />
```

Comment:
Note that, while I added this component in the previous PR, I'm not deleting it here -- I'm sure we will reuse it in the future.